.github/workflows/hpos-build-and-e2e-tests-daily.yml

```diff
@@ -46,6 +46,14 @@ jobs:
               working-directory: plugins/woocommerce
               run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js
 
+            - name: Run Playwright E2E tests on chrome-new-editor.
```

Comment:
Similar comment to the above `chrome-new-editor` is very generic, we should be more specific.

Reply:
Also we should make a similar change here as I mentioned below, where this is moved up to its own run entirely.